blocksuite/framework/block-std/src/gfx/viewport-element.ts

```diff
@@ -46,7 +46,7 @@ export class GfxViewportElement extends WithDisposable(ShadowlessElement) {
     }
   `;
 
-  private readonly _hideOutsideBlock = requestThrottledConnectedFrame(() => {
+  private readonly _hideOutsideBlock = () => {
```

Comment:
Will this affect viewport panning perf?

Reply:
No, because it has been throttled in `_refreshViewport`. No need to call this method again.